docs/changelog.md

```diff
@@ -10,6 +10,9 @@ sidebar_label: Changelog
 
 - Added [Bolt over WebSocket](/connect-to-memgraph/websocket.md) support to
   Memgraph. [#384](https://github.com/memgraph/memgraph/pull/384)
+- Added [user-defined Memgraph magic
+  functions](/cypher-manual/functions/#user-defined-memgraph-magic-functions).
```

Comment:
I checked the link, it works. However I am not sure why `functions.md` doesn't work instead of `functions` in the URL. In older changelog items it is used like that:
```
Added `CONFIGS` and `CREDENTIALS` options to [Kafka streams](/reference-guide/streams/overview.md/#kafka).
```

Reply:
I would like to know that as well, maybe @g-despot knows?